src/components/form-field/form-field/form-field.scss

```diff
@@ -175,13 +179,19 @@
   border: var(--sbb-form-field-border-width) var(--sbb-form-field-border-style)
     var(--sbb-form-field-border-color);
   border-radius: var(--sbb-form-field-border-radius);
-  height: var(--sbb-form-field-height);
+  min-height: var(--sbb-form-field-min-height);
+  max-height: var(--sbb-form-field-max-height);
   background-color: var(--sbb-form-field-background-color);
 
   :host([data-input-focused][data-focus-origin='keyboard']) & {
     @include sbb.focus-outline;
   }
 
+  // Prevents page content from shifting on textarea focus
+  :host([data-input-focused][data-input-type='textarea']) & {
+    margin-block-end: calc(var(--sbb-form-field-border-width) / 2 * -1);
+  }
+
```

Comment:
:(

Reply:
I konw, I also don't like them and I'm open to possible alternative solutions. To answer your question, it is just a textarea problem due to the `height: fit-content` to enable the resize; as long as you have a fixed height like in any other case, the label/content shifting doesn't occur.

Reply:
I would also ask @jeripeierSBB or @kyubisation, what do you think about it?

Reply:
Understood the problem :/ 
The only alternative I see is to move the focus border to a `::before` element. We already do something similar in other components

Reply:
Could solve it according to your suggestion @TomMenga . Thanks :)